src/app/year-calendar/year-calendar.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="dp-year-calendar-container">
+  <dp-calendar-nav
+      [label]="navLabel"
+      [showLeftNav]="showLeftNav"
+      [showLeftSecondaryNav]="showSecondaryLeftNav"
+      [showRightNav]="showRightNav"
+      [showRightSecondaryNav]="showSecondaryRightNav"
+      [isLabelClickable]="componentConfig.isNavHeaderBtnClickable"
+      [showGoToCurrent]="shouldShowCurrent()"
```

Comment:
please don't use getter functions in template, use variables instead, this was implemented in 2.6.0 cross all components

Reply:
But this is the same way in `month-calendar/month-calendar.component.html`, isn't it?

Reply:
this might be something I missed, thanks for pointing out. Will fix on the month calendar.